lib/baseFactory.js

```diff
@@ -113,6 +112,31 @@ class BaseFactory {
     }
 
     /**
+     * Remove a Model
+     * @method remove
+     * @param  {Object}    config               Config object
```

Comment:
The type of `config` should be `mixed`? Since config can be just a string of id instead of an object.